src/components/modeTodo/modeTodo.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prop-types */
+/* import { useState } from 'react/cjs/react.development'; */
+import './modeTodo.css';
+
+const Mode = (props) => {
+/*  const [edit, setMode] = useState(''); */
```

Comment:
**[REQUIRED]** Please remove all commented code from your codebase, this will make your code looker cleaner.